Content.Server/Construction/Components/TabletopComputerBoardComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Construction.Components
+{
+    [RegisterComponent, ComponentProtoName("TabletopComputerBoardComponent")]
+    public sealed partial class TabletopComputerBoardComponent : Component
```

Comment:
Is there any reason why TabletopComputerBoard needs to be it's own bespoke component? This leads to a *massive* amount of copy-paste in construction graph prototypes.

Reply:
I tried adding this to an existing ComputerComponent, but I couldn't get it to work, so I had to use this option. I agree that this should have been avoided, but I couldn't...